document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java

```diff
@@ -22,18 +23,20 @@ public class PostgresUtils {
   private static final String DOT_STR = "_dot_";
   private static final String DOT = ".";
 
-  private static final Set<String> OUTER_COLUMNS = Set.of(CREATED_AT, ID, UPDATED_AT);
+  public static final Set<String> OUTER_COLUMNS =
+      new TreeSet<>(List.of(ID, CREATED_AT, UPDATED_AT));
```

Comment:
Any reason for preferring a TreeSet over the normal set?

Reply:
No specific reason, wanted to have generated fields in query same pattern. But, that's not the mandatory requriement.